README.md

```diff
@@ -181,6 +184,29 @@ Note that our CI system checks that code formatting is consistent, and the build
 
 Also note that we run some `clang-tidy` linting over the code as well (see `.clang-tidy` for the list of rules enforced).  You can run `./scripts/tidy.sh` over the code before committing to ensure you haven't added any of the common problems we check for (Note: `./scripts/tidy.sh` requires the exitence of a `compile_commands.json` database.  You can generate this file by running `bear make` instead of just `make`.  The `bear` tool is installable on Ubuntu-based systems with `apt-get install bear`, and on macOS with `brew install bear`).
 
+Using the Node.js Bindings
+--------------------------
+
+The Node.js bindings are still under construction. We are working on building binaries for as many environments as possible, but they may not all be available yet. The first functionality that we are exposing is `route`, but we plan on exposing more of the tyr actor functions. Right now, the input and the output are both strings - THAT WILL CHANGE. We plan on ingesting and producing protobufs.
+
+The Node.js bindings provide read-only access to the routing engine. You can install the Node.js bindings via npm install valhalla or from this repository either via
+
+`$npm install`
+which will check and use pre-built binaries if they're available for this release and your Node version, or via
+
+`$npm install --build-from-source`
+to always force building the Node.js bindings from source.
+
+Example of using in a node project:
+```js
+var valhalla = require('valhalla');
+var router = new valhalla.Actor(configString);
+var hersheyRequest = '{"locations":[{"lat":40.546115,"lon":-76.385076,"type":"break"}, {"lat":40.544232,"lon":-76.385752,"type":"break"}],"costing":"auto"}';
```

Comment:
So... I've been wondering: As these are Node bindings.. Would it be nicer if we would actually use a Node object `{"locations":[{"lat":40.546115,"lon":-76.385076,"type":"break"}, {"lat":40.544232,"lon":-76.385752,"type":"break"}],"costing":"auto"}` as an input parameter instead of its string? 🤔

Reply:
I agree (with the caveat that I don't know how to do it). However, I was thinking we were maybe planning on having it ingest protobuf ultimately, and didn't want to spend time figuring out how to get Node objects to work if that was the plan?